quarkus/runtime/src/test/java/org/keycloak/provider/quarkus/ConfigurationTest.java

```diff
@@ -110,57 +111,57 @@ public void testCamelCase() {
 
     @Test
     public void testEnvVarPriorityOverPropertiesFile() {
-        putEnvVar("KC_SPI_HOSTNAME_DEFAULT_FRONTEND_URL", "http://envvar.com");
-        assertEquals("http://envvar.com", initConfig("hostname", "default").get("frontendUrl"));
+        putEnvVar("KC_SPI_HOSTNAME_DEFAULT_FRONTEND_URL", "http://envvar.unittest");
```

Comment:
some of the URLs we used here were actually reachable. Imo we should use some kind of pattern like this here instead.